types.d.ts

```diff
@@ -28,6 +28,15 @@ export interface IBaseSvgRendererProps extends IMoleculeToSVGOptions {
   onBondEnter?: (bondId: number, event: MouseEvent<SVGElement>) => void;
   onBondLeave?: (bondId: number, event: MouseEvent<SVGElement>) => void;
   onBondClick?: (bondId: number, event: MouseEvent<SVGElement>) => void;
+
+  autoCrop?: boolean;
+  autoCropMargin?: number;
+
+  label?: {
+    fontFamily?: string;
+    fontSize?: string;
+    value?: string;
+  };
```

Comment:
sub-objects are not usual in React props. And here I find it inconsistent with the other options like bond highlight that are separated in multiple props.